src/sbt-test/sbt-cake/testrunner/start-server.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+pushd target/universal/stage
+./bin/testrunner &
+echo $! > server.pid
```

Comment:
This is only ever written to, but never read. Was the intention to use it in `stop-server.sh`?